powertools-sqs/src/main/java/software/amazon/lambda/powertools/sqs/SqsUtils.java

```diff
@@ -98,6 +98,16 @@ public static void overrideSqsClient(SqsClient client) {
         SqsUtils.client = client;
     }
 
+    /**
+     * Provides ability to set default {@link S3Client} to be used by utility.
```

Comment:
By default the S3Client is instantiated via {@link S3Client#create()}. This method provides the ability to override the S3Client with your own custom version.